plugin/lsp.vim

```diff
@@ -37,3 +37,19 @@ command! -range LspDocumentRangeFormat call lsp#ui#vim#document_range_format()
 command! LspImplementation call lsp#ui#vim#implementation()
 command! LspTypeDefinition call lsp#ui#vim#type_definition()
 command! -nargs=0 LspStatus echo lsp#get_server_status()
+
+nnoremap <expr> <plug>(lsp-code-action) [lsp#ui#vim#code_action(),''][1]
+nnoremap <expr> <plug>(lsp-definition) [lsp#ui#vim#definition(), ''][1]
+nnoremap <expr> <plug>(lsp-document-symbol) [lsp#ui#vim#document_symbol(),''][1]
+nnoremap <expr> <plug>(lsp-document-diagnostics) [lsp#ui#vim#diagnostics#document_diagnostics(),''][1]
+nnoremap <expr> <plug>(lsp-hover) [lsp#ui#vim#hover#get_hover_under_cursor(),''][1]
+nnoremap <expr> <plug>(lsp-next-error) [lsp#ui#vim#signs#next_error(),''][1]
+nnoremap <expr> <plug>(lsp-previous-error) [lsp#ui#vim#signs#previous_error(),''][1]
+nnoremap <expr> <plug>(lsp-references) [lsp#ui#vim#references(),''][1]
+nnoremap <expr> <plug>(lsp-rename) [lsp#ui#vim#rename(),''][1]
+nnoremap <expr> <plug>(lsp-workspace-symbol) [lsp#ui#vim#workspace_symbol(),''][1]
+nnoremap <expr> <plug>(lsp-document-format) [lsp#ui#vim#document_format(),''][1]
+vnoremap <expr> <plug>(lsp-document-format) [lsp#ui#vim#document_range_format(),''][1]
```

Comment:
Do you prefer this is `<plug>(lsp-document-range-format)` ?

Reply:
I'm either way. We can add `lsp-document-range-format` if we get feedback.